packages/cli/src/commands/hydrogen/deploy.ts

```diff
@@ -82,7 +81,7 @@ export default class Deploy extends Command {
     }),
     preview: Flags.boolean({
       description:
-        'Deploys to the Preview environment. Overrides --env-branch and Git metadata.',
+        'Deploys to the Preview environment. Overrides Git metadata.',
```

Comment:
This previously stated that it `Overrides --env-branch`, but looking at the original code in this file [a little further down](https://github.com/Shopify/hydrogen/blob/main/packages/cli/src/commands/hydrogen/deploy.ts#L404-L435) it appears that this wasn't actually the case (and I think we should also remove the claim that this overrides the Git metadata too?)

Reply:
yes - remove `--env-branch` comment

`overrides Git metadata` - @frandiox Does `hydrogen deploy preview` actually overrides git metadata? I don't see its doing that in code either. Running `npx shopify hydrogen deploy --preview` just creates a preview deploy with no changes to user's repo (which is expected). I think the metadata here is referring to oxygen specific metadata.